model-optimizer/install_prerequisites/install_prerequisites.sh

```diff
@@ -83,18 +83,130 @@ elif [[ "$OSTYPE" == "darwin"* ]]; then
     python3 -m pip install --upgrade pip
 fi
 
+install_latest_ov() {
+    if $2; then
+        sudo -E "$1" -m pip install openvino
```

Comment:
If I remember right the `"` is redundant.

Reply:
No, it is bash code-style requirements.